Cargo.toml

```diff
@@ -24,6 +24,7 @@ slog = "2.1"
 thread_local = "1"
 take_mut = "0.2.0"
 crossbeam-channel = "0.5"
+serde = "1"
```

Comment:
I think we would need a new cfg feature here instead.

Reply:
Not sure I can follow.

Should i128 support be enabled automatically if available, or manually?

If it should be enabled automatically, I guess we can add a build script that sets a cfg feature, like [serde does it](https://github.com/serde-rs/serde/blob/9c39115f827170f7adbdfa4115f5916c5979393c/serde/build.rs#L59). Is this what you meant?

If Rust support < 1.26 is not a concern, it could just be enabled always, wdyt?

Reply:
> Is this what you meant?

That would work, yes.

We just don't want to add a new dependency (serde) for something unreleated (u128).